apollo-runtime/src/main/java/com/apollographql/apollo/internal/subscription/RealSubscriptionManager.java

```diff
@@ -106,6 +135,10 @@ public void removeOnStateChangeListener(@NotNull OnStateChangeListener onStateCh
   }
 
   void doSubscribe(Subscription subscription, SubscriptionManager.Callback callback) {
+    if (state == State.STOPPING || state == State.STOPPED) {
+      callback.onError(new ApolloSubscriptionException("Illegal state: " + state.name()));
```

Comment:
Can we add more descriptive message. As it will be reported to user so we want to make it more user friendly

Reply:
All set. Let me know what you think of the current message.